keras/backend/theano_backend.py

```diff
@@ -201,6 +201,18 @@ def is_keras_tensor(x):
     return hasattr(x, '_keras_history')
 
 
+def is_native_tensor(x):
```

Comment:
InputLayer will have some metadatas once https://github.com/fchollet/keras/pull/7066 is merged.